TYPO3.Neos/Documentation/HowTos/ConfigureImageGeneration.rst

```diff
@@ -0,0 +1,28 @@
+==========================
```

Comment:
This documentation should be added to the media package documentation instead of the Neos documentation. See https://github.com/neos/neos-development-collection/tree/master/TYPO3.Media/Documentation (neos-media.readthedocs.org)